src/config.inc

```diff
@@ -1,10 +1,19 @@
 # Build platform (use one of AUTO, Unix, OSX_Universal, MSVC, Cygwin, MinGW)
 BUILD_ENV = AUTO
 
-# Variables you may want to override
-#CXXFLAGS = -Wall -O0 -g -Werror -Wno-long-long -Wno-sign-compare -Wno-parentheses -Wno-strict-aliasing -pedantic
+# Specify the C++ standard
 #CXXFLAGS = -std=c++11
```

Comment:
Doesn't that depend on which version of which compiler the developer is using? This is in the public repo so anyone could be building this.